bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -115,9 +115,13 @@ spec:
             if [[ -f /var/run/openvswitch/flows.sh ]]; then
               echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Adding br0 if it doesn't exist ..." 2>&1
               /usr/bin/ovs-vsctl --may-exist add-br br0 -- set Bridge br0 fail_mode=secure protocols=OpenFlow13
+              echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Renaming recovery file in case we fail during restoration ..." 2>&1
+              mv /var/run/openvswitch/flows.sh /var/run/openvswitch/flows-old.sh
               echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Created br0, now adding flows ..." 2>&1
-              sh -x /var/run/openvswitch/flows.sh
+              sh -x /var/run/openvswitch/flows-old.sh
               echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Done restoring the existing flows ..." 2>&1
+              rm -f /var/run/openvswitch/flows-old.sh
+              echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Deleting recovery file after restore ..." 2>&1
```

Comment:
don't need that message either; if the `rm` fails it will print an error, and if it doesn't then we know it worked

Reply:
ack, I'll also remove the -f, as otherwise it'll just pass